docs/components/Navigation.tsx

```diff
@@ -65,16 +65,20 @@ export function Navigation() {
         <NavItem href="/tutorials/getting-started-with-create-keystone-app">
           Getting started
         </NavItem>
+        <NavItem href="/tutorials/embedded-mode-with-nextjs-frontend">
+          Embedding Keystone in NextJS
+        </NavItem>
       </Section>
       <Section label="Guides">
         <NavItem href="/guides/keystone-5-vs-keystone-next">Keystone 5 vs Next</NavItem>
         <NavItem href="/guides/cli">Command Line</NavItem>
         <NavItem href="/guides/relationships">Relationships</NavItem>
         <NavItem href="/guides/document-fields">Document Fields</NavItem>
+        <NavItem href="/guides/hooks">Hooks</NavItem>
+        <NavItem href="/guides/modes">Modes</NavItem>
```

Comment:
I would put modes under Roadmap not into Guides... wdyt?

Reply:
Hmmm, under `roadmap` feels like we’re giving it too much import by placing so high in the nav.

I'm inclined to leave it under guides for the time being, as it is a form of guidance, definitely not a tut, and has nothing to do with API ref.

Think in the not too distant future we may need a small section for `concepts`. We could talk about this, and other high-level ideas that form the backdrop of a Keystone authoring experience. 🤔  Maybe there's a page for Typescript & Keystone, that kinda thing.

Reply:
Per our conversation in standup, moving this into a new CONCEPTS nav section.